weave-js/src/components/Form/Select.tsx

```diff
@@ -177,6 +178,7 @@ export const Select = <
       return {
         ...baseStyles,
         padding: '6px 0',
+        ...props.menuListStyle,
```

Comment:
I added this new prop to the common Select component because I noticed we don't have the option to override the styles.

Reply:
ahhh I like that wayy better - changed to use `MenuList`!